src/helpers/ens.ts

```diff
@@ -36,12 +37,72 @@ export async function getSpaceUriFromContentHash(id) {
 }
 
 export async function getSpaceUriFromTextRecord(id) {
+  const REGISTRAR_ABI = [
+    {
```

Comment:
Can you use this format for ABI https://github.com/snapshot-labs/snapshot.js/blob/master/src/sign/eip1271.ts#L15 ? It's much more compact

Reply:
You can find it on Etherscan